app/services/funnel/registration/add_mfa.rb

```diff
@@ -3,14 +3,33 @@
 module Funnel
   module Registration
     class AddMfa
-      def self.call(user_id, mfa_method, analytics)
+      def self.call(user_id, mfa_method, analytics, threatmetrix_attrs = {})
```

Comment:
Is there anywhere we're not expecting `threatmetrix_attrs` to be passed? If not, I'd suggest not making it optional, to prevent unexpected usage.

```suggestion
      def self.call(user_id, mfa_method, analytics, threatmetrix_attrs)
```